pytorch_lightning/trainer/lr_finder.py

```diff
@@ -115,10 +119,12 @@ def lr_find(self,
 
         """
         if num_accumulation_steps is not None:
-            rank_zero_warn("Argument `num_accumulation_steps` has been deprepecated"
-                           " since v0.7.6 and will be removed in 0.9. Please"
-                           " set trainer argument `accumulate_grad_batches` instead.",
-                           DeprecationWarning)
+            rank_zero_warn(
+                "Argument `num_accumulation_steps` has been deprepecated"
+                " since v0.7.6 and will be removed in 0.9. Please"
+                " set trainer argument `accumulate_grad_batches` instead.",
```

Comment:
```suggestion
                "Argument `num_accumulation_steps` has been deprecated since v0.7.6 and will be removed in 0.9."
                " Please set trainer argument `accumulate_grad_batches` instead.",
```